include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -1266,6 +1274,109 @@ namespace internal
 
       const auto reference_cell = dof_handler_fine.get_fe(0).reference_cell();
 
+      std::unique_ptr<internal::MatrixFreeFunctions::HangingNodes<dim>>
+                                             hanging_nodes;
+      std::vector<std::vector<unsigned int>> lexicographic_mappings;
```

Comment:
Do we use this name elsewhere? I think we should try to keep somewhat uniform names for the same thing within the library, e.g. `lexicographic_numberings` or `lexicographic_numbering_per_fe`.

Reply:
The name originates from https://github.com/dealii/dealii/blob/acabf316ecc4cd92835ebef7547ed59588aabce5/include/deal.II/matrix_free/hanging_nodes_internal.h#L747

(old implementation)